src/w3d/math/quaternion.cpp

```diff
@@ -313,7 +311,9 @@ void Slerp_Setup(const Quaternion &p, const Quaternion &q, SlerpInfoStruct *sler
         slerpinfo->flip = false;
     }
 
-    if (1.0f - cos_t < SLERP_EPSILON) {
+    constexpr double slerp_epsilon = 0.001;
```

Comment:
This is a `double` in World Builder. Whether or not that makes any practical difference, it does not hurt to leave it original.